### DISCUSSION TITLE
Sorting exports

### DISCUSSION THREAD

**Discussion by hugo-dlb** [trivago/prettier-plugin-sort-imports#124] [Ideas]
Thanks for developing this sweet plugin!
Are there plans to sort exports as well? It is pretty common to wrap third-party libraries and re-export many components, this would be a possible use case.

**Comment by rebellious-developer** [comment]
I'd love to have this as an optional feature as well. Maybe even a `prettier-plugin-sort-exports` since the rules for sorting exports would be somewhat different/trickier than imports (which can be natural sorted fairly easily).

**Comment by ekwoka** [comment]
I'd like this. Maybe I'll take a stab at it.

Would be nice for keeping barrel files clean and clear.
